source/client/benchmark_client_impl.h

```diff
@@ -22,13 +22,9 @@
 
 #include "api/client/options.pb.h"
 
-#include "common/statistic_impl.h"
```

Comment:
With this PR, you have to include files in the `source/` directory using full path. This is reflected in the bullet 2 in the description. Previous commit had the same two files included twice using full and partial path, in the `source/client/benchmark_client_impl.h`. You can not do it any more. It will fail to compile.

Reply:
Got it, I missed that while reviewing. Thank you.